public/src/js/profile.js

```diff
@@ -15,7 +15,8 @@ async function populateProfile(user) {
 
     if (user) {
         document.getElementById('fullName').innerHTML = user.displayName;
-        document.getElementById('small_profile_image').setAttribute('src', userPhoto);
+        if (userPhoto)
+            document.getElementById('small_profile_image').setAttribute('src', userPhoto);
```

Comment:
It is good practice to use `{ ... }` even with one-line `if` statements:
```suggestion
        if (userPhoto) {
            document.getElementById('small_profile_image').setAttribute('src', userPhoto);
        }
```